### ISSUE TITLE
Req: Load video only on click, not on scroll.

### ISSUE THREAD

**Issue by wplit** [vime-js/vime#85]
It could be useful to have an option to switch between the default lazy load to instead an 'on click' load.

Sort of like.. https://css-tricks.com/lazy-load-embedded-youtube-videos/

Meaning when the element is scrolled to, only the poster image is fetched at first (or if vime-youtube, vime-vimeo allowed custom poster image, nothing would be needed to be fetched from those providers at first) 

Then only when the user clicks the element, then the video is actually loaded from youtube itself. (Useful for pages with lots of videos where a lot of the videos won't actually ever be played by site visitors but would still load as they scroll via lazy loading.

**Comment by mihar-22** [comment]
Good idea, maybe a property on the player called `loadStrategy` that could have values such as `immediate`, `lazy` and `interact` or something like that.
